### ISSUE TITLE
Empty black box on iOS

### ISSUE THREAD

**Issue by tjvantoll** [Yermo/nativescript-mapbox#1]
Hey @EddyVerbruggen,

I tried out this plugin today and it works great on Android, but on iOS I’m unfortunately not seeing a map—all I see is a black box.

![img_0837](https://cloud.githubusercontent.com/assets/544280/11783097/3b9604e4-a243-11e5-8195-819e782f87aa.PNG)

I’ve tried your demo and my own custom app and I get the same. Any idea what might be up here? The logs tell me that I have permissions and `mapbox.show` doesn’t log any errors, so I’m not sure what’s up.

Thanks.

**Comment by tjvantoll** [comment]
Oh, one completely unrelated note. This code on the docs:

``` js
mapbox.hasFineLocationPermission().then(
    function(granted) {
      // if this is 'false' you probably want to call 'requestFineLocationPermission' now
      console.log("Has Location Permission? " + result);
    }
);
```

`result` is not defined here. I’m guessing you meant to use `granted` in the `console.log()` call.

**Comment by EddyVerbruggen** [comment]
Hey TJ, thanks for the report. It was a bit buried by all other stuff in my mailbox, but I just stumbled on it again and took a moment to fix it.

Turned out the Mapbox guys changed the location of the resource files a bit, so the plugin could no longer load the map. That's what you get when you don't pin to a specific SDK version. The plugin now pins to a version it is known to work well with (should have done that in the first place).

The doc is also fixed, thanks for noticing that!
